### DISCUSSION TITLE
Disabling scripting in PDF-Forms (XFA,...)

### DISCUSSION THREAD

**Discussion by DavidLaun** [mozilla/policy-templates#958] [Q&A]
Hi, I hope here is the right place to post this question.

One of our customers wants to disable all kinds of scripting in the buil-in PDF viewer.

We are planning on setting the  following "preference":  `"pdfjs.enableScripting": { "Value": false, "Status": "locked" },`

When switching to the new v102 ESR version we noticed this info in the release notes:

> Firefox PDF viewer now supports filling more forms (e.g., XFA-based forms, used by multiple governments and banks).[...]

Will the above preference setting also effect those kind of forms?

Thanks, David

**Comment by mkaply** [comment]
You can disable XFA by setting

pdfjs.enableXfa

to false.

**Reply by DavidLaun** [reply]
Thank you, we will use this setting.
